borg/archiver.py

```diff
@@ -1399,7 +1440,7 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='ARCHIVE',
                                type=location_validator(archive=True),
                                help='name of archive to create (must be also a valid directory name)')
-        subparser.add_argument('paths', metavar='PATH', nargs='+', type=str,
+        subparser.add_argument('paths', metavar='PATH', nargs='*', type=str,
```

Comment:
Unfortunately changing nargs from '+' to '*' here may trigger a bug in pythons argparse (https://bugs.python.org/issue15112) under special circumstances.
If an optional argument (``--anything``) is placed on the commandline between the positional arguments `location` and `paths`, path-arguments are not populated correctly.